pluginExamples/EnlightenPlugin.py

```diff
@@ -171,6 +171,7 @@ def get_configuration_obj(self):
             name = self.name, 
             fields = self._fields,
             is_blocking = self.is_blocking,
+            block_enlighten = self.block_enlighten,
```

Comment:
My recollection is that is_blocking basically blocks *the plugin* — it cannot receive / process new Requests until it finishes processing the last one.

block_enlighten blocks *ENLIGHTEN* — it can't move forward with processing the current spectrum, or doing anything, until the Request is complete.